de.tu_bs.cs.isf.e4cf.core/src/de/tu_bs/cs/isf/e4cf/core/db/TableServiceImp.java

```diff
@@ -0,0 +1,496 @@
+package de.tu_bs.cs.isf.e4cf.core.db;
+
+import java.lang.reflect.Field;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TableServiceImp extends TableUtilities implements ITableService {
+
+	/**
+	 * Method to create a table from a given String tableName and the attributes
+	 * from the class Column.
+	 * 
+	 * @param pPath      String the path of the database
+	 * @param pDbName    String the name of the database
+	 * @param tableName  String the name of the table
+	 * @param attributes Class which contains the attributes and their data types
+	 * @throws SQLException
+	 */
+	@Override
+	public void createTable(final String pPath, final String pDbName, final String tableName,
+			final Column... attributes) throws SQLException {
+		final Connection con = DatabaseFactory.getInstance().getDatabase(pPath, pDbName);
+		final Statement stmt = con.createStatement();
+		boolean primaryKey = false;

```

Comment:
evtl. "isPrimaryKey" für besseres Verständnis im code

Reply:
Es ist ein Flag, damit man weiß, ob es Primary Keys in der Tabelle gibt. Diese sollten dann später weiter behandelt werden